File: C#-OOP/HelloWorld.cs
using CSharp_OOP.Factory;
using CSharp_OOP.Models;
using CSharp_OOP.Services;
using System;

namespace CSharp_OOP
{
    public class HelloWorld
    {
		public static void run(String arg)
		{ 
			Message message = new Message(arg);
			AbstractPrinterFactory factory = AbstractPrinterFactory.GetFactory(); 
			IPrinter printer = factory.GetPrinter(); 
			message.PrintOut(printer);
		}
	}
}

File: PHP-OOP/src/HelloWorld.php
<?php
namespace app;

use app\factory\AbstractPrinterFactory;
use app\models\Message;
/*
 * program utama :)
 */
class HelloWorld 
{ 
	public static function run($arg) 
	{ 
		$message = new Message($arg); 
		$factory = AbstractPrinterFactory::getFactory(); 
		$printer = $factory->getPrinter(); 
		$message->printOut($printer); 
	} 
}


